sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetReadBenchmark.scala

```diff
@@ -1,339 +0,0 @@
-/*
```

Comment:
@maropu . Since you are merging ParquetReadBenchmark and OrcReadBenchmark benchmarks, let's remove [OrcReadBenchmark](https://github.com/apache/spark/blob/master/sql/hive/src/test/scala/org/apache/spark/sql/hive/orc/OrcReadBenchmark.scala).

Reply:
I feel we still need to `OrcReadBenchmark` to compare native `orc` with `Hive built-in orc`?

Reply:
`Hive built-in orc` is also `data source`. If we can move `DataSourceReadBenchmark.scala` from `org.apache.spark.sql` to `org.apache.spark.sql.hive.orc`, we can add `Hive built-in orc` into this benchmark, too.

Reply:
ok, thanks!